Resources/Locale/en-US/advertisements/vending/yarrrdrobe.ftl

```diff
@@ -0,0 +1,5 @@
+advertisement-yarrrdrobe-1 = Eye patches 50% off.
+advertisement-yarrrdrobe-2 = Plunder some booty while you dress as a beauty!
+advertisement-yarrrdrobe-3 = Who needs an extra leg anyways?
+advertisement-yarrrdrobe-4 = Come get yer supplies while you last!
+advertisement-yarrrdrobe-5 = Oi! Land lubber get o'er here and plunder me!
```

Comment:
Small rewrite.
```suggestion
advertisement-yarrrdrobe-5 = Oi, landlubber! Get o'er here and plunder me!
```

Reply:
Other suggestions:
X marks the spot for incredible deals!
Best outfits this side of Tortuga.
Brace the main sale!